argus/introspection/cloud/windows.py

```diff
@@ -397,3 +397,17 @@ def get_user_flags(self, user):
                 "{0} {1}".format(remote_script, user),
                 command_type=util.POWERSHELL_SCRIPT_BYPASS)
             return stdout.strip()
+
+    def get_swap_status(self):
+        """Get whether the swap memory is enabled or not.
+
+        :returns: True if swap memory is enabled, False if not.
+        :rtype: bool
+        """
+        swap_query = (r"'HKLM:\SYSTEM\CurrentControlSet\Control\Session"
+                      r" Manager\Memory Management'")
+        cmd = r'(Get-ItemProperty {}).PagingFiles'.format(swap_query)
```

Comment:
This is just a cosmetic thing, but you could have the `swap_query` without single quotes and add them in the `cmd` line . `(Get-ItemProperty '{}')...`